File: Assignment_2/program4.c
#include<stdio.h>
void Display(int iNo,int iFrequency)
{
    int i=0;

    for(i=0;i < iFrequency;i++)
    {
        printf("%d\t",iNo);
    }
  
}
int main()
{
    int iValue=0;
    int iCount=0;

    printf("enter number:");
    scanf("%d",&iValue);
    printf("enter frequency:");
    scanf("%d",&iCount);
    Display(iValue,iCount);
    return 0;
}

File: Assignment_21/program4.java
import java.util.Scanner;

class LogicX
{
    void DisplayFactors(int iNo)
    {
        int iCnt=0;
        int iCount=0;
        
        for(iCnt=1;iCnt<iNo/2;iCnt++) 
        {
            if(iNo%iCnt==0)
            {
                iCount=iCount+iCnt;
            }
        }
        System.out.println(iCount);
    }
}

class program4
{
    public static void main(String A[])
    {
        int iValue=0;
        Scanner sobj=new Scanner(System.in);

        System.out.println("Enter the number to find its factors :");
        iValue=sobj.nextInt();

        LogicX lobj=new LogicX();

        lobj.DisplayFactors(iValue);
    }
}